src/resources/items/ItemForm/ItemFormToolbar.tsx

```diff
@@ -49,11 +51,18 @@ const ItemFormToolbar = (props: Props): React.ReactElement => {
     }
   }, [])
 
-  const transformResource = (
+  const transformResource = async (
     data: Record<string, any>
-  ): Record<string, any> => {
+  ): Promise<Record<string, any>> => {
     const { catCave, catCode, catHandling, ...rest } = data
-    return rest
+
```

Comment:
Hello @salony75way - how about only pulling in the fields from the `batch` if both of them are null?   If one of them is not-null then we know the backend is pulling in the extra data, and we don't need this trick.

Reply:
> Hello @salony75way - how about only pulling in the fields from the `batch` if both of them are null? If one of them is not-null then we know the backend is pulling in the extra data, and we don't need this trick.

Hi @IanMayo, Its not clear to me, Can you please elaborate a bit more?